pkg/azure/sdk_handler.go

```diff
@@ -290,8 +290,8 @@ func (h *AzureSDKHandler) DeleteSecurityRule(ctx context.Context, nsgName string
 	return nil
 }
 
-// GetVnetAddressSpaces returns a map of location to address space for all virtual networks (VNets) with a given prefix.
-func (h *AzureSDKHandler) GetVNetsAddressSpaces(ctx context.Context, prefix string) (map[string][]string, error) {
+// Returns a map of location to address space for all virtual networks (VNets) with a given prefix.
+func (h *AzureSDKHandler) GetVnetsWithMatchingPrefixAddressSpaces(ctx context.Context, prefix string) (map[string][]string, error) {
```

Comment:
This new name confuses me -- why the change?

Reply:
Ah, I see. Maybe instead: `GetAllVnetAddressSpaces`?

Reply:
I need to work on choosing better variable and function names. I'll go with this and add function doc to it.